contracts/libraries/LibSortValidatorsByBeaconOld.sol

```diff
@@ -0,0 +1,223 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.9;
+
+import { LibArray } from "./LibArray.sol";
+import { IndexMap, LibIndexMap } from "./LibIndexMap.sol";
+import { governanceValidatorFilter, standardValidatorFilter, nonStandardValidatorFilter } from "../utils/Filters.sol";
+
+library LibSortValidatorsByBeaconOld {
+  using LibArray for *;
+  using LibIndexMap for *;
```

Comment:
Please indicate explicitly instead of `*`.

Reply:
fixed.